h2o-r/tests/testdir_algos/automl/runit_automl_model_selection.R

```diff
@@ -211,7 +244,8 @@ automl.model_selection.suite <- function() {
     test_modeling_plan_full_syntax,
     test_modeling_plan_minimal_syntax,
     test_modeling_steps,
-    test_exclude_algos_is_applied_on_top_of_modeling_plan
+    test_exclude_algos_is_applied_on_top_of_modeling_plan,
+    test_monotone_constraints_can_be_passed_as_algo_parameter,
```

Comment:
@seb-h2o for Py you have two more tests 
``` 
    test_algo_parameter_can_be_applied_only_to_a_specific_algo,
    test_cannot_set_unauthorized_algo_parameter
```
just in case it was forgotten.

Reply:
it was not really forgotten... always wondering if Py and R should have the same tests.
Usually I cover REST API + Py client in Py tests, whereas I just cover R client in R tests...

Reply:
`test_algo_parameter_can_be_applied_only_to_a_specific_algo` could be added to R though

Reply:
added R test above (and discovered a bug from it)